pages/common/kill.md

```diff
@@ -1,12 +1,28 @@
 # kill
 
-> Sends a signal to a process.
-> Mostly used for stopping processes.
+> Sends a signal to a process, usually related to stopping the process.
+> All signals except for SIGKILL and SIGSTOP can be intercepted by the process to perform a clean exit.
 
-- Kill the process:
+- Terminate a program using the default SIGTERM (terminate) signal:
 
 `kill {{process_id}}`
 
-- List signal names:
+- List available signal names (to be used without the `SIG` prefix):
 
 `kill -l`
+
+- Terminate a program using the SIGHUP (hang up) signal. Many daemons will reload instead of terminating:
+
+`kill {{1|-HUP}} {{process_id}}`
```

Comment:
Shouldn't this be `-1`? And if so, wouldn't it be more concise to have `-{{1|HUP}}` instead?